lib/gamnit/flat.nit

```diff
@@ -184,10 +198,14 @@ redef class App
 	end
 
 	# Draw the whole screen, all `glDraw...` calls should be executed here
-	protected fun frame_core_draw(display: GamnitDisplay) do frame_core_flat display
+	protected fun frame_core_draw(display: GamnitDisplay)
+	do
+		frame_core_world_sprites display
+		frame_core_ui_sprites display
+	end
 
-	# Draw sprites in `sprites` and `ui_sprites`
-	protected fun frame_core_flat(display: GamnitDisplay)
+	# Draw world sprites from `sprites`
+	protected fun frame_core_world_sprites(display: GamnitDisplay)
```

Comment:
you should explain what is the deal withe the various *_sprites : what is the point of each category and what the used has to do